fuzzers/007-timing/.gitignore

```diff
@@ -1,9 +1,2 @@
-/.Xil
-/design/
-/design.bit
-/design.bits
-/design.dcp
-/usage_statistics_webtalk.*
-/vivado*
-/specimen_*
-build_speed
+build
+
```

Comment:
Extra newline.

Reply:
This issue is in lots of places. I haven't paid a lot of attention to it